azure/sdk-identity-resources-storage/README.md

```diff
@@ -50,14 +50,24 @@ Use the [Azure Cloud Shell](https://shell.azure.com) to create and get client se
     }
```

Comment:
When I ran this there was no 'name' field in the output. Was that unique to me for some reason or we would expect nobody would see 'name' any longer and we should remove it from the expected output?

Reply:
There is no `name` attribute any more. Instead, there is `displayName`. Check out the standard output below:
![image](https://user-images.githubusercontent.com/4126750/150063350-97ad6d2f-d46e-4520-9af6-ceb031bc53d6.png)

Reply:
Do you know if there is a reason we see different output for the same command? This is what I see when running that command:
![image](https://user-images.githubusercontent.com/6243776/150064442-3cbf7ef9-8c50-4091-965b-8c9743b4afb5.png)


I was proposing that we delete line 47 under the assumption that other users would see what I see. But if some users will see it and others won't that sadly makes this more confusing no matter what we put.

Reply:
Ok... which version of the CLI are you running? I can see that you're either on a Linux box or CloudShell? My version of the CLI is `az --version`
azure-cli: 2.32.0
core: 2.32.0
telemetry: 1.0.6
Extensions: 
cosmosdb-preview: 0.5.0
msal: 1.16.0
azure-mgmt-resource: 20.0.0

Reply:
@Heaths, would you know?

Reply:
Off hand, no, but I know the CLI has been making some changes. As the warning shows in the first screenshot, the `name` was going to be deprecated. Why they changed it I'm not sure.